subsys/bluetooth/adv_prov/providers/gap_appearance.c

```diff
@@ -10,7 +10,7 @@
 #include <bluetooth/adv_prov.h>
 
 
-static int get_data(struct bt_data *ad, const struct bt_le_adv_prov_adv_state *state,
+static int get_data(struct bt_data *d, const struct bt_le_adv_prov_adv_state *state,
```

Comment:
Was the rename `ad` -> `d` intentional? (You can just resolve this comment if yes.)